### ISSUE TITLE
Can't start hammer++ using wine

### ISSUE THREAD

**Issue by EngiFort** [ficool2/HammerPlusPlus-Issue-Tracker#385]
`
0024:err:module:import_dll Library tier0.dll (which is needed by L"Z:\\home\\engifort\\.local\\share\\Steam\\steamapps\\common\\Team Fortress 2\\bin\\hammerplusplus.exe") not found
0024:err:module:import_dll Library vstdlib.dll (which is needed by L"Z:\\home\\engifort\\.local\\share\\Steam\\steamapps\\common\\Team Fortress 2\\bin\\hammerplusplus.exe") not found
0024:err:module:LdrInitializeThunk Importing dlls for L"Z:\\home\\engifort\\.local\\share\\Steam\\steamapps\\common\\Team Fortress 2\\bin\\hammerplusplus.exe" failed, status c0000135
`
And it doesn't start. I've searched on winetricks for this vstdlib.dll and tier0.dll but didn't find it. It seems to be a hl2 thing?

**Comment by colleen05** [comment]
Have you found a solution? I'm having the same issue.

**Comment by EngiFort** [comment]
yes! I followed the instructions of this [thread](https://tf2maps.net/threads/how-to-install-the-hammer-editor-on-linux.42473/) on tf2maps.net and got hammer++ working. But remember to put the hammerplusplus files in yours tf2 bin/ folder and not the downloaded depot one

**Comment by colleen05** [comment]
alright, thanks! i'll try to get this working with CS:GO.
